Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -282,11 +282,13 @@ public class CRTClientEngine: HTTPClient {
                                         }
 
                                         var hasMoreChunks = true
```

Comment:
Changes here is to match how `ChunkedReader` is used by `URLSessionHTTPClient` via `ChunkedStream`. Before the change, the logic was slightly different.